### ISSUE TITLE
About the code of finetuning on the specific dataset

### ISSUE THREAD

**Issue by momo1986** [SysCV/sam-hq#11]
Thanks for this siginificant and interesting work.
Open-source work is helpful to the developer
In the paper, the authors write "In particular, we use both global semantic context and local fine-grained features by fusing SAM’s mask decoder features with early and late feature maps from its ViT encoder. During training, we freeze the entire pre-trained SAM parameters, while only updating our HQ-Output Token, its associated three-layer MLPs, and a small feature fusion block.". **Is there a demo to run the training code or should we reproduce them by ourselves?**

Thanks & Regards!
Momo

**Comment by lkeab** [comment]
Hi, thanks for watching HQ-SAM. We are organizing the training/finetuning codes and will release it after it's ready. Please stay tuned!

**Comment by unriddler** [comment]
> 

Cool. Looking forward to the training code.

**Comment by xiawei20161308104** [comment]
> Hi, thanks for watching HQ-SAM. We are organizing the training/finetuning codes and will release it after it's ready. Please stay tuned!

looking forward~！

**Comment by astro-fits** [comment]
looking forward~！

**Comment by momo1986** [comment]
looking forward~！

**Comment by lkeab** [comment]
Thanks, we plan to release it at the beginning of July.

**Comment by yahooo-m** [comment]
looking forward~

**Comment by zbbb31** [comment]
Looking forward, fine-tuning on this powerful tool would be excited!

**Comment by lkeab** [comment]
Thanks. It will be ready this week.

**Comment by lkeab** [comment]
Hi, we released the [training codes for SAM tuning](https://github.com/SysCV/sam-hq#hq-sam-tuning-and-hq-seg44k-data).
